migrate/v0_16/kava.go

```diff
@@ -59,8 +59,11 @@ func migrateKavaAppState(appState genutiltypes.AppMap, clientCtx client.Context)
 		var genState v015committee.GenesisState
 		v15Codec.MustUnmarshalJSON(appState[v015committee.ModuleName], &genState)
 
+		var pricefeedGenState v015pricefeed.GenesisState
+		v15Codec.MustUnmarshalJSON(appState[v015pricefeed.ModuleName], &pricefeedGenState)
+
 		// replace migrated genstate with previous genstate
-		appState[v015committee.ModuleName] = v16Codec.MustMarshalJSON(v016committee.Migrate(genState))
+		appState[v015committee.ModuleName] = v16Codec.MustMarshalJSON(v016committee.Migrate(genState, pricefeedGenState))
```

Comment:
Any recommendations for handling missing states? Seems like missing module states just results in silently skipping over those modules, not sure if we would want to do the same here -- would a panic or warning be ideal here if pricefeed genesis state is missing? 

Also doesn't seem too ideal to have to pass the pricefeed genesis state from here through all of the following functions, but doesn't seem like there's another option to get pricefeed markets other than hard coding them in.

Reply:
since committee migration depends on the price feed one, i agree that it might be good to panic here if it is missing.
I think passing genState to the migrate function should be okay? Cosmos migration also do this for x/bank and I don't see any other good options.